python/cuml/cuml/linear_model/linear_regression_mg.pyx

```diff
@@ -57,13 +52,11 @@ cdef extern from "cuml/linear_model/ols_mg.hpp" namespace "ML::OLS::opg" nogil:
 
 
 class LinearRegressionMG(MGFitMixin, LinearRegression):
-
-    def __init__(self, **kwargs):
-        super(LinearRegressionMG, self).__init__(**kwargs)
-
     @cuml.internals.api_base_return_any_skipall
     def _fit(self, X, y, coef_ptr, input_desc):
-
+        cdef int algo = (
+            Algo.EIG if self.algorithm == "auto" else Algo.parse(self.algorithm)
+        )
```

Comment:
Why don't we fallback to SVD under specific circumstances like in the single GPU estimator?

Reply:
The SVD fallback is written in pure python, not the native SVD version (which only supports 1D y). The multigpu LinearRegression already only supports dimensions that the native calls work for, hence no fallback.